src/transcription/prerecorded/options.rs

```diff
@@ -24,6 +24,9 @@ pub struct Options {
     search: Vec<String>,
     replace: Vec<Replace>,
     keywords: Vec<Keyword>,
+    filler_words: bool,
+    smart_format: bool,
+    paragraphs: Option<bool>,
```

Comment:
Can you change `filler_words` and `smart_format` to `Option<bool>`, so they don't get serialized unless the user requests them?